apps/site/assets/ts/models/__tests__/alert-test.ts

```diff
@@ -44,6 +45,17 @@ export const beforeNow = () => {
   ];
 };
 
+export const threeDaysFromNow = () => {
+  const now = Date.now();
+  const five_minutes = 300_000;
+  return [
+    [
+      activePeriodDateFormatted(now - five_minutes),
+      activePeriodDateFormatted(add(now, { days: 3 }).getMilliseconds())
```

Comment:
Wondering if it is worth making `activerPeriodDateFormatted` take in Dates and milliseconds.  The first thing it does is cast the milliseconds to a `Date` object.  Then no need to call `getMilliseconds` on these calls